### ISSUE TITLE
`function-calc-no-unspaced-operator` and Sass functions

### ISSUE THREAD

**Issue by deepfriedmind** [stylelint/stylelint#933]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

An issue where `function-calc-no-unspaced-operator` is unnecessarily triggered in Sass.

> What CSS is needed to reproduce this issue?

```scss
.foo {
  margin-top: rem-calc(map-get($grid-column-gutter, small));
}
```

> Which version of stylelint are you using?

`5.0.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin via gulp-postcss

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SCSS functions.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
Expected single space after "-" operator (function-calc-no-unspaced-operator)
```

So the issue stems from that the Sass function contains the word "calc" (`rem-calc` is a default function in the Foundation framework, i.e. something widely used) and that `map-get` contains a hyphen.

**Comment by davidtheclark** [comment]
@deepfriedmind : If you can file a PR with at least failing tests that illustrate the need, that would really help speed up this fix.

**Comment by davidpustai** [comment]
:+1: to this one. It doesn't take dash in variables as minus, it does count dash in function name as minus.
